accounts/templates/account.html

```diff
@@ -2,41 +2,30 @@
 
 <html>
 
-<head>
-
-<link rel="stylesheet" href="{% static 'style.css' %}">
-
-</head>
-
-
-<body>
-	{% if user.is_authenticated %}
-		<h1>Welcome, {{ user.get_username }}</h1>
-		
-		<p>
-		<a href="{% url 'home_page' %}"> Back to JoBoard </a>
-		</p>
-		
-		<p>
-		<a href="{% url 'logout' %}"> Log Out </a>
```

Comment:
why we need this page?
wht not dierct the user to the search page after login sucssusfuly?